### PR TITLE
Add Bitly connector

### PR DESCRIPTION
**Pull request by SkNuwanTissera**
## Purpose
Add Bitly connector

## Documentation
https://dev.bitly.com/docs

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

## Environment
Ballerina SL Beta 3

### CODE REVIEW

File: openapi/bitly/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+target
+tests
```
Comment (RolandHewage) [inline openapi/bitly/.gitignore:2]: You can remove this line

File: openapi/bitly/Package.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Connects to [Bitly API v4.0.0](https://dev.bitly.com/api-reference) from Ballerina
+
+## Package overview
+The `ballerinax/bitly` is a [Ballerina](https://ballerina.io/) connector for Bitly API.
+This package provides the capability to access c API.
```
Comment (RolandHewage) [inline openapi/bitly/Package.md:5]: Please add the name of the API here.

**Review by RolandHewage — COMMENTED** [review]
Please remove the  `module-ballerinax-googleapis.sheets` section. Minor changes are requested. Rest is fine.

**Review by RolandHewage — CHANGES_REQUESTED** [review]
Please remove the `module-ballerinax-googleapis.sheets section`. Minor changes are requested. Please fix them. Rest is fine.

File: openapi/bitly/.gitignore (hunk 1 above)
Comment (SkNuwanTissera) [reply openapi/bitly/.gitignore]: Hi, Did the suggested changes. 
/Thanks

File: openapi/bitly/Package.md (hunk 2 above)
Comment (SkNuwanTissera) [reply openapi/bitly/Package.md]: Hi, Did the suggested changes. 
/Thanks

**Comment by SkNuwanTissera** [comment]
Hi, Did the suggested changes.  @Chuhaa @RolandHewage 
/Thanks

**Review by Chuhaa — APPROVED** [review]

**Review by RolandHewage — COMMENTED** [review]
Please regenerate the connector using the latest tool.

**Review by RolandHewage — APPROVED** [review]

**Pull request merged**